data/announcements/example.yaml

```diff
@@ -0,0 +1,28 @@
+---
+### Example announcement
+#
+# Leave this example announcement in place. It serves as a template
+# and as documentation by example.
+#
+# 🛈  Announcement changes require approval from @kubernetes/steering-committee
+#
+#
+apiVersion: docs.k8s.io/v1alpha1
+kind: Announcement
+spec:
+  priority: 0 # Largest number wins tie break
+  # dateConstraints are optional
+  dateConstraints:
+    notAfter: 2020-04-01T00:00:00
+    notBefore: 2020-01-01T00:00:00
```

Comment:
suggest making both of these required; currently, a typo in this field name would make an announcement perpetual accidentally, right?

Reply:
also, unless there's prior art in the website config, would avoiding negatives in field names be clearer?

straw-man:
```
start: 2020-01-01T00:00:00
stop: 2020-04-01T00:00:00
```

Reply:
Naming wise, how about `startDateTime` and `endDateTime`?

> suggest making both of these required; currently, a typo in this field name would make an announcement perpetual accidentally, right?

If the steering committee want an open-ended announcement that's fine by me, or (for example) setting no start date because an announcement should go live as soon as it is merged.

I could force a site build to fail if:
- `endDateTime` is not set
- `endDateTime` doesn't parse as a date.
- `startDateTime` is set and doesn't parse as a date.

How about that?

Reply:
I think an explicit end is always good, avoids typos doing the wrong thing, and is easy to make effectively open-ended by picking a far-future datetime

Reply:
> Naming wise, how about startDateTime and endDateTime?

Sure, that's easier than the negative field names (at least for my tired end-of-year brain)

Reply:
could make the name even easier to read, 
startTime
endTime